app.py

```
@@ -118,10 +197,15 @@ def process_attendance_data(df, id_number):
         'summary': attendance_summary
     }
 
+
 def main():
     st.title("Student Attendance Report Generator")
-    # Instructions for CSV format
-    # st.subheader("CSV File Format")
+    # Add Twilio credentials in sidebar
+    st.sidebar.header("SMS Configuration")
+    twilio_account_sid = st.sidebar.text_input("Twilio Account SID", type="password")
```

Ruff (0.8.2)

Local variable `twilio_account_sid` is assigned to but never used

Remove assignment to unused variable `twilio_account_sid`

(F841)